app/server/routes/core.ts

```diff
@@ -42,4 +45,28 @@ router.get("/robots.txt", (_, res: Response) => {
   res.contentType("text/plain").send(accessList);
 });
 
+router.get("/help-centre/sitemap.txt", async (_, res: Response) => {
```

Comment:
Hi Kelvin.
I had the idea the the sitemap needed to be placed at the base of the path. But maybe I'm wrong?

Reply:
Moved it now.